src/main/webapp/app/shared/http/file-uploader.service.ts

```diff
@@ -14,7 +14,7 @@ type Options = {
 @Injectable({ providedIn: 'root' })
 export class FileUploaderService {
     // NOTE: this list has to be the same as in FileResource.java
-    acceptedFileExtensions = 'png,jpg,jpeg,svg,pdf,zip';
+    acceptedFileExtensions = 'png,jpg,jpeg,svg,pdf,zip,odt,ods,odp,odg,odf,doc,docx,csv,xls,xlsx,ppt,pptx,pages,rtf,pages-tef,numbers,key';
```

Comment:
Would it make sense to save this as a proper array of extensions and then generate both
- this string of `acceptedFileExtensions`, and
- another string used for the tooltip text from it?

Then the translation key `fileLimitations` would only contain the size limit and the shown tooltip text is combined from the generated list and the translation. That would remove a possible source of inconsistency after future changes to this list.

Reply:
I agree, the file names are language independent, so we don't even need to translate them

Reply:
Implemented.